### PR TITLE
Missing live response limitations + command status

### PR DESCRIPTION
**Pull request by marinaevteeva**
Added missing limitations:
- Only one running live response API call on one machine.
- Commands must be in specific order.
- PutFile doesn't have a response even when successful.

Clarified on potential command statuses, which are different from action statuses

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/run-live-response.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ Runs a sequence of live response commands on a device
 
 7. Multiple live response commands can be run on a single API call. However, when a live response command fails all the subsequent actions won't be executed.
 
+8. Multiple live response sessions can't be executed on the same machine (if live response action is already running, subsequent requests are responded with HTTP 400 - ActiveRequestAlreadyExists)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-live-response.md]:
```suggestion
8. Multiple live response sessions can't be executed on the same machine (if live response action is already running, subsequent requests are responded to with HTTP 400 - ActiveRequestAlreadyExists).
```

File: microsoft-365/security/defender-endpoint/run-live-response.md [hunk 2]
```diff
@@ -171,6 +173,8 @@ POST https://api.securitycenter.microsoft.com/api/machines/1e5bc9d7e413ddd7902c2
 
 Here's an example of the response.
 
+Possible values for each command status are "Created", "Completed", "Failed".
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-live-response.md]:
```suggestion
Possible values for each command status are "Created", "Completed", and "Failed".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @marinaevteeva, kindly commit the suggested changes. Many thanks.

**Comment by marinaevteeva** [comment]
@JohanFreelancer9 Thank you. Applied the suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**